pkg/discovery/worker/group_discovery_policy_cvs.go

```diff
@@ -89,24 +95,26 @@ func resolveCVSPolicyTargets(
 func createCVSPolicy(
 	worker *k8sEntityGroupDiscoveryWorker,
 	policyBinding *repository.TurboPolicyBinding) (*proto.GroupDTO_SettingPolicy_, error) {
+	pbName := fmt.Sprintf("%v/%v", policyBinding.GetNamespace(), policyBinding.GetName())
+	glog.V(4).Infof("Create container vertical scale policy from policy bind %v", pbName)
 	spec := policyBinding.GetContainerVerticalScaleSpec()
 	if spec == nil {
-		return nil, fmt.Errorf("the ContainerVerticalScaleSpec is nil")
+		return nil, fmt.Errorf("the ContainerVerticalScaleSpec is nil in policy binding %v", pbName)
 	}
 
 	settings := group.NewSettingsBuilder()
 
 	aggressiveness := spec.Settings.Aggressiveness
 	if aggressiveness != nil {
 		val := PercentileAggressiveness[(string(*aggressiveness))]
-		glog.V(2).Infof("Aggressiveness %f", val)
+		glog.V(4).Infof("Aggressiveness:  %f", val)
```

Comment:
Should we include the policy name with these to add more context? Perhaps it's unnecessary 🤷‍♂️ we don't process these in parallel, do we?

Reply:
We do not process that in parallel.